src/lib/server/blog/posts.ts

```diff
@@ -120,3 +128,22 @@ export function getOfficerByGhUsername(ghUsername: string): Officer | null {
   );
   return officer ?? null;
 }
+
+/**
+ * Performs the following post-processing steps on the HTML content of a
+ * blog post
+ * 1. Removes the "disabled" attribute from any checkboxes in the document
+ * 2. Wraps all .task-list-item contents to make them css selectable
+ * @param html The HTML content of the blog post
+ * @returns The modified HTML string
+ */
+export function _postProcessHTML(html: string) {
```

Comment:
The underscore prefix may be omitted for this exported function. It is not required to make a distinction that this is exported exclusively to be tested. 
```suggestion
export function postProcessHTML(html: string) {
```